test/01_basic/cibuildwheel_basic_test.py

```diff
@@ -1,16 +1,20 @@
 import os
-import utils
+import sys 
+
+sys.path.append(os.path.dirname(os.path.dirname(__file__)))
+
+from shared import utils
```

Comment:
I guess this changed is required by:
1. `pytest` direct invocation
2. the removal of `PYTHONPATH` fix in `bin/run_test.py` (which is probably removed because of the change for `pytest` didn't require `bin/run_test.py` to still fix the `PYTHONPATH `)

Reply:
yes

Reply:
This `sys.path.append` should then be in `conftest.py`, I believe?